### DISCUSSION TITLE
Form submissioin

### DISCUSSION THREAD

**Discussion by Claymost** [realstoman/vuejs-tailwindcss-portfolio#32] [Q&A]
Hey, beautiful project. Doesn't seem at this point that the form submission is hooked up to anything. Is the intention for us to use this in conjunction with some other backend framework / API?

**Comment by realstoman** [comment]
Yes exactly, it's just the UI at the moment, but you can use any other framework like Node.js or Laravel so you can send the data to a database or you can simply use a mailing service like MailChimp to get the data from the user directly into your email.
